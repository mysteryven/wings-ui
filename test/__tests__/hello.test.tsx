import * as renderer from "react-test-renderer";
import * as React from "react";
import {Icon} from '../../lib/index';

describe('Components export', ()=>{
    it('Icon exports successfully', ()=> {
        const json = renderer.create(<Icon name="car"/>).toJSON();
        expect(json).toMatchSnapshot();
    })
});