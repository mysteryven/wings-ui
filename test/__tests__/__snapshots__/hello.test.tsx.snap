// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components export Icon exports successfully 1`] = `
<svg
  className="wen-design-svg"
>
  <use
    xlinkHref="#car"
  />
</svg>
`;
